.gitignore

```diff
@@ -1,2 +1,3 @@
 *~
 patches/bitbake/*.APPLIED
+*.sw*
```

Comment:
I know was likely to deal with issues you were having, but does it belong in this RFC PR?

Reply:
sure, will drop it